.clang-format

```
@@ -0,0 +1,41 @@
+---
+ColumnLimit: 120
```

```suggestion
ColumnLimit: 0
```
Thanks for the feedback.
Would this value be better?
Or 1024 ?